app/components/ReferralModal.tsx

```
@@ -0,0 +1,142 @@
+                        disabled={isSubmitting || !referralCode.trim()}
+                        className="min-h-11 w-full rounded-xl bg-lavender-500 py-2 text-sm font-medium text-white disabled:text-white/30 transition-colors hover:bg-lavender-600 disabled:cursor-not-allowed disabled:dark:bg-white/5"
+                    >
+                        {isSubmitting ? "Submitting..." : "Submit"}
+                    </button>
+
+                    <button
+                        type="button"
+                        onClick={handleSkip}
+                        className="min-h-11 w-full rounded-xl bg-accent-gray py-2 text-sm font-medium text-gray-900 transition-colors hover:bg-accent-gray/80 dark:bg-white/5 dark:text-white dark:hover:bg-white/10"
+                    >
+                        I don't have a referral code
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)